### PR TITLE
Add direct write file to avoid moving docker mounted files

### LINKED ISSUE #1154: Request failed with status code 502
**Describe the problem**
Installed Netbird by using getting-started-with-zitadel.sh on a OVH VPS but after accessing console I'm gettin 502 error 
obraz

netbird-management-1 container keep crashing with following error message
`2023-09-19T19:46:44Z INFO management/server/telemetry/app_metrics.go:161: enabled application metrics and exposing on http://0.0.0.0:8081
Error: failed to write out store encryption key: rename /etc/netbird/.2772331671management.json /etc/netbird/management.json: device or resource busy
`

**To Reproduce**

Follow Quickstart guide

**Expected behavior**
Working console

### PR DESCRIPTION
**Pull request by mlsmaycon**
## Issue ticket number and link
fixes #1154
### Checklist
- [x] Is it a bug fix
- [ ] Is a typo/documentation fix
- [ ] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: util/file.go [hunk 1]
```diff
@@ -54,6 +56,63 @@ func WriteJson(file string, obj interface{}) error {
...
+	}
+
+	targetFile, err := os.Create(file)
+	if err != nil {
+		return nil, err
+	}
+	//no:lint
+	err = targetFile.Chmod(0640)
+	if err != nil {
+		_ = targetFile.Close()
```
Comment (pappz) [inline util/file.go:115]: We should close the file in all cases independently from the err of the chmod

**Review by pappz — CHANGES_REQUESTED** [review]

File: util/file.go (hunk 1 above)
Comment (braginini) [reply util/file.go:115]: It is closed in the DirectWriteJson function

Comment (pappz) [reply util/file.go:115]: I think no. Because we schedule the close with defer in line 72. and in case of chmod err we never reach the 72. line.

Comment (pappz) [reply util/file.go:115]: Ahh, you are right

**Review by pappz — CHANGES_REQUESTED** [review]

**Comment by pappz** [comment]
Something is not correct. It provide invalid json file.
This is the end of my file:
```
            "Scope": "openid",
            "UseIDToken": false,
            "RedirectURLs": null
        }
    },
    "PKCEAuthorizationFlow": null
}ow": null
}
```

**Review by pappz — APPROVED** [review]

**Pull request merged**